### ISSUE TITLE
Warn if files are being left behind by tests

### ISSUE THREAD

**Issue by gregcaporaso** [caporaso-lab-graveyard/clout#14]
It would be useful if Clout could raise a warning of some type if tests are leaving files behind. This is useful when running tools like QIIME on a cluster. If runs of the test suite leave files with different names (e.g., randomly generated filenames) behind, those can build up to use a lot of disk space over many runs.

**Comment by jairideout** [comment]
Good idea- it would also be nice to see if processes are being left behind (e.g. in QIIME, poller.py can hang).

**Comment by gregcaporaso** [comment]
Agreed.

**Comment by jairideout** [comment]
Was looking into this a bit. [This thread](http://stackoverflow.com/q/5893748) has some interesting ideas. We could look for files that were modified/created since the test suite started running, or could make a list of all files on the system before running the tests and do a diff with the list of files afterwards.

Does anyone have additional ideas for how to tackle this? I'm also interested in catching processes that are being left behind.
